bindings/python/setup.py

```diff
@@ -25,15 +31,46 @@ def query_pkgconfig(cmd):
 
 def append_libbson_flags(module):
     pc_path = 'libbson-1.0'
-    install_dir = os.environ.get('LIBBSON_INSTALL_DIR')
+    install_dir = os.environ.get('MONGO_LIBBSON_DIR')
     if install_dir:
-        libdirs = glob.glob(os.path.join(install_dir, "lib*"))
-        if len(libdirs) != 1:
-            warnings.warn("Unable to locate {}".format('libbson-1.0.pc'))
+        # Handle the copy-able library file if applicable.
+        if COPY_LIBBSON:
+            if platform == "darwin":
+                lib_file = "libbson-1.0.0.dylib"
+            elif platform == "linux":
+                lib_file = "libbson-1.0.so.0"
+            else:  # windows
+                lib_file = 'bson-1.0.dll'
+            lib_dir = 'bin' if os.name == 'nt' else 'lib*'
```

Comment:
I suggest making `os.name == 'nt'` a variable to ease readability. I'm also more familiar with `sys.platform == 'win32'` which is what we use in other project to detect windows.

Reply:
Will do